aait/backend/group-2B/Application.Tests/Features/Authentication/Handlers/Queries/LoginUserRequestHandlerTest.cs

```diff
@@ -106,4 +113,38 @@ public async Task Invalid_IncorrectUsernameLoginUser()
         result.IsSuccess.ShouldBeFalse();
         result.Value.ShouldBeNull();
     }
+
+    [Fact]
+    public async Task Invalid_InvalidUsernameLogin()
```

Comment:
This test must be testing against non existent username instead of saying invalid username. Because the username is valid but it may not exist.